Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData.Tests/OpenFlightsDataProviderTests.cs

```diff
@@ -6,7 +6,7 @@
 using Ibistic.Public.OpenAirportData.OpenFlightsData;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace Ibistic.Public.OpenAirportData.Tests
+namespace Ibistic.Public.OpenAirportData.Tests2
```

Comment:
I don't see a reason to change this namespace.

Reply:
should be ok now